src/nuclide_header.F90

```diff
@@ -198,6 +192,15 @@ function library_path_c(type, name) result(path) bind(C, name='library_path')
       character(kind=C_CHAR), intent(in) :: name(*)
       type(C_PTR) :: path
     end function
+
+    subroutine nuclide_calculate_urr_xs_c(use_mp, i_nuclide, i_temp, E) &
```

Comment:
Let's just make this `nuclide_calculate_urr_xs` so that you don't need to specify a special name on the `bind(C)`